### PR TITLE
Added Victoriametrics Single to Marketplace-App OneClickApp

### PR DESCRIPTION
**Pull request by denisgolius**

### CODE REVIEW

File: victoriametrics-single/victoriametrics_documentation.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+- [OpenTSDB](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-opentsdb-compatible-agents) on port :4242
+- Influx (telegraph) on port :8089 tcp/udp
+
+See more details and examples in [official documentation](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html).
+
+### UI
+
+VictoriaMetrics provides a [User Interface (UI)](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#vmui) for query troubleshooting and exploration. The UI is available at `http://your_droplet_public_ipv4:8428/vmui`. It lets users explore query results via graphs and tables.
+
+To check it, open the following in your browser `http://your_droplet_public_ipv4:8428/vmui` and then enter `vm_app_uptime_seconds` to the Query Field to Execute the Query.
```
Comment (hmorris3293) [inline victoriametrics-single/victoriametrics_documentation.md:74]: Will need to remove the "droplet" references as that is specific to Digital Ocean. You can use "Linodes" in place for droplets

File: victoriametrics-single/victoriametrics_documentation.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Run the following command to query and retrieve a result from VictoriaMetrics Single with `curl`:
+
+```bash
+curl -sg http://your_linodes_ip_address_ipv4:8428/api/v1/query_range?query=vm_app_uptime_seconds | jq
+```
+
+### Accessing
+
+Once the Droplet is created, you can use DigitalOcean's web console to start a session or  SSH directly to the server as root:
```
Comment (hmorris3293) [inline victoriametrics-single/victoriametrics_documentation.md:84]: remove droplet & Digital Ocean reference

**Review by hmorris3293 — COMMENTED** [review]
Hi @denisgolius  Thank you for the submission! I've made a few suggestions to the documentation. I've also tested the installation script and it appears that the vmsingle service fails to start, so it appears that the app doesn't work as expected. Can you please confirm that the installation script installs the VictoriaMetics service correctly?

**Comment by denisgolius** [comment]
Hi @hmorris3293 !
Thanks for your feedback! I have fixed StackScript  and documentation file.

**Comment by denisgolius** [comment]
Hey @hmorris3293 ! Any updates?

**Comment by hmorris3293** [comment]
hi @denisgolius  All looks good, tested and confirmed it works as expected!

**Pull request merged**